.github/workflows/build.yaml

```diff
@@ -22,17 +22,31 @@ jobs:
 
       - uses: docker/setup-qemu-action@v2
 
+      - run: docker context create build
+
       - uses: docker/setup-buildx-action@v2
+        with:
+          endpoint: build
 
-      - uses: docker/login-action@v2
+      - name: Login docker hub
+        uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_PUSH_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PUSH_TOKEN }}
+      
+      - name: Login registry.smtx.io
+        uses: docker/login-action@v2
+        with:
+          registry: registry.smtx.io
+          username: ${{ secrets.HARBOR_PUSH_USERNAME }}
+          password: ${{ secrets.HARBOR_PUSH_TOKEN }}
```

Comment:
can we use https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategy for different registry?

Reply:
It will run multi jobs.
we want to build image one time, and push image to two docker hub(public docker hub and smtx.io docker hub)